tests/std/tests/GH_000639_nvcc_include_all/env.lst

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+PM_COMPILER="nvcc" PM_CL="--x cu -Xcompiler /EHsc,/nologo,/W4,/WX,/openmp"
+RUNALL_CROSSLIST
+PM_CL="-Xcompiler /MT"
+PM_CL="--debug -Xcompiler /MTd,-D_ITERATOR_DEBUG_LEVEL=2"

```

Comment:
Let's use `/` uniformly for pass-through arguments:
```suggestion
PM_CL="-Xcompiler /MT"
PM_CL="--debug -Xcompiler /MTd,/D_ITERATOR_DEBUG_LEVEL=2"
```
Should we also pass `/Od` (the "don't optimize" flag) to both of these?

Reply:
This is taken from the internal env.lst as is. Nvcc think we're referring to the `D:` drive if I don't use a - when passing this flag.

Reply:
I can pass `/Od` but again, I was just taking the `env.lst` from the internal harness.